src/components/Calendar/ExportCalendar.js

```diff
@@ -98,7 +95,7 @@ const getFirstClass = (date, time) => {
 
 // getExamTime returns the start and end datetime of an exam
 //  Ex: ("Mon Jun 7 10:30-12:30pm", "2019") -> [[2019, 6, 7, 10, 30], [2019, 6, 7, 12, 30]]
-const months = { Mar: 3, Apr: 4, Jun: 6, Jul: 7, Aug: 8, Sep: 9, Nov: 11, Dec: 12 };
+const months = { Mar: 3, Jun: 6, Aug: 8, Sep: 9, Dec: 12 };
```

Comment:
Why are we removing these specific months from this object?

Reply:
I went through the Quarterly Academic Calendars on reg.uci.edu and found that final exams always took place in March (Winter), June (Spring), July (Summer I), August (Summer 10wk), September (Summer II), and December (Fall), so April and November aren't needed. Good catch, though, since I also erroneously removed the entry for July.